genbadge/main.py

```diff
@@ -24,6 +24,7 @@
 OUTFILE_BADGE_HELP = ("An alternate SVG badge file to write to. '-' is supported and means <stdout>. Note that in this "
                       "case no other message will be printed to <stdout>. In particular the verbose flag will have no "
                       "effect.")
+NAME_HELP = ("An alternate SVG badge text name to displag on the left hand side of the badge.")
```

Comment:
```suggestion
NAME_HELP = ("An alternate SVG badge text name to display on the left hand side of the badge.")
```

Reply:
Also : if setting an empty value on this to remove the left side of the badge works, we should add it here